### ISSUE TITLE
Unable to assign host, no host available

### ISSUE THREAD

**Issue by TheBarzFR** [theqvd/theqvd#82]
Hi again,
I have this following error when i trying starting VM: 
"Action not accomplished for all elements, Unable to assign host, no host available"
The status of the node is "Stopped".

Do i forget something ?
How do i starting the node ?
Thanks.

**Comment by jzeaherra** [comment]
Hi,

 `systemctl start qvd-hkd`

That should start the node. You can in parallel:
`tail -f /var/log/qvd.log`

to see if there is any problem with your config.

**Comment by TheBarzFR** [comment]
qvd-hkd.service - LSB: Example initscript
   Loaded: loaded (/etc/init.d/qvd-hkd; bad; vendor preset: enabled)
   Active: active (exited) since mer. 2019-02-20 17:16:04 CET; 17h ago
     Docs: man:systemd-sysv-generator(8)
  Process: 4035 ExecStop=/etc/init.d/qvd-hkd stop (code=exited, status=0/SUCCESS)
  Process: 4048 ExecStart=/etc/init.d/qvd-hkd start (code=exited, status=0/SUCCESS)
    Tasks: 0
   Memory: 0B
      CPU: 0

févr. 20 17:16:03 uxvvm010 systemd[1]: Starting LSB: Example initscript...
févr. 20 17:16:04 uxvvm010 qvd-hkd[4048]: given is experimental at /usr/lib/qvd/lib/perl5/site_perl/5.26.1/QVD/HKD/
févr. 20 17:16:04 uxvvm010 qvd-hkd[4048]: when is experimental at /usr/lib/qvd/lib/perl5/site_perl/5.26.1/QVD/HKD/A
févr. 20 17:16:04 uxvvm010 qvd-hkd[4048]: when is experimental at /usr/lib/qvd/lib/perl5/site_perl/5.26.1/QVD/HKD/A
févr. 20 17:16:04 uxvvm010 systemd[1]: Started LSB: Example initscript.

**Comment by TheBarzFR** [comment]
I stopped all services. Removed logs. Restart services.
Now i have this logs for the two nodes:
2019/02/21 11:17:00 8664 ERROR /usr/lib/qvd/lib/perl5/site_perl/5.26.1/QVD/HKD.pm 367 - Host IP address is outside of the network range reserved for hosts (IP: x.x.91.126, range: 90.90.90.0-90.90.90.1)
2019/02/21 11:20:43 8747 ERROR /usr/lib/qvd/lib/perl5/site_perl/5.26.1/QVD/HKD.pm 367 - Host IP address is outside of the network range reserved for hosts (IP: x.x.91.126, range: 90.90.90.0-90.90.90.1)
2019/02/21 11:22:54 8851 ERROR /usr/lib/qvd/lib/perl5/site_perl/5.26.1/QVD/HKD.pm 367 - Host IP address is outside of the network range reserved for hosts (IP: x.x.91.123, range: 90.90.90.0-90.90.90.1)

But in the QVD config files the network range are x.x.90.1 to x.x.91.253

**Comment by jzeaherra** [comment]
Your logs suggests your network config is wrong. You need to have a bridge for VM's network. That bridge is assigned the gateway ip for your defined network range.  In this example from the docs:
```
# qa4 config set tenant_id=-1,key=vm.network.ip.start,value=10.3.15.50
# qa4 config set tenant_id=-1,key=vm.network.netmask,value=24
# qa4 config set tenant_id=-1,key=vm.network.gateway,value=10.3.15.1
# qa4 config set tenant_id=-1,key=vm.network.dns_server,value=10.3.15.254
# qa4 config set tenant_id=-1,key=vm.network.bridge,value=qvdnet0
```
( http://docs.theqvd.com/docs/en/4.1/QVDQuickInstallationGuideLinuxUbuntu.html#_network_configuration)
You need a bridge called qvdnet0 up and running with ip 10.3.15.1.

**Comment by TheBarzFR** [comment]
Hi,
In fact i already done that:

vm.network.bridge	qvdnet0
vm.network.dns_server 	x.90.91.251	
vm.network.domain	
vm.network.firewall.nat.iface
vm.network.gateway 		x.90.90.254
vm.network.ip.end 		x.90.91.253
vm.network.ip.start		x.90.90.1

I would connect the vm machine in the same network as the nodes.
Is it possible ?

**Comment by TheBarzFR** [comment]
Hi
I done like on the doc the settings. It's the same trouble. 
Now i can't do the qa4 commands as root. The CLI says : "Wrong credentials - Login again"
any ideas ,
